mesheryctl/doc/doc.go

```diff
@@ -157,6 +167,14 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 	if err := printOptions(buf, cmd, name); err != nil {
 		return err
 	}
+
+	// If command has screenshot present
+	if picLine != "" {
+		buf.WriteString("## Screenshots\n\n")
+		buf.WriteString(picLine)
+		buf.WriteString("\n\n")
```

Comment:
Looks like an opportunity to capture a caption.

Example: "Display the list of all available applications."

https://twitter.com/mesheryio/status/1542556666733899776

Reply:
Ok, another line to be added then in the logic 🙂

Reply:
👨‍🎤